cloudflare/schema_cloudflare_notification_policy.go

```diff
@@ -25,11 +25,26 @@ func resourceCloudflareNotificationPolicySchema() map[string]*schema.Schema {
 			Required: true,
 		},
 		"filters": {
-			Type:     schema.TypeMap,
+			Type:     schema.TypeSet,
```

Comment:
you still want this as a `TypeList` otherwise it won't be addressable as `filters.0...`
```suggestion
			Type:     schema.TypeList,
```